tests/ParseSimpleTest.php

```diff
@@ -8,20 +8,22 @@ final class ParseSimpleTest extends TestCase
 {
   public function testParseSimple(): void
   {
-    $rtf = file_get_contents("tests/rtf/hello-world.rtf");
+    $rtf = file_get_contents(__DIR__ ."/rtf/hello-world.rtf");
+
+    
     $document = new Document($rtf);
     $this->assertTrue(true);
   }
 
   public function testParseSimpleHtml(): void
   {
-    $rtf = file_get_contents("tests/rtf/hello-world.rtf");
+    $rtf = file_get_contents(__DIR__ ."/rtf/hello-world.rtf");
     $document = new Document($rtf);
     $formatter = new HtmlFormatter();
     $html = $formatter->Format($document);    
 
     $this->assertEquals(
-      '<p><span style="font-family:Calibri;font-size:15px;">Hello, world.</span></p>',
+      '<p><span style="font-family:Calibri;font-size:15px;">Hello, world.</span></p><p></p>',
```

Comment:
Was the extra \<p>\</p> already being generated and causing the test to fail?

Reply:
Yes already being generated, it's also the case in our existing tests in our app. Perhaps something to look into at some point